AwesomeProject/styles/AboutMe.styles.js

```diff
@@ -1,18 +1,17 @@
 import {StyleSheet} from 'react-native';
```

Comment:
styles should be in the same place with file where it used. In your case this file should be in AboutMe folder